apps/frontend/src/store/snackbar.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  Module,
+  VuexModule,
+  Mutation,
+  Action,
+  getModule
+} from 'vuex-module-decorators';
+import Store from '@/store/store';
+
+export interface ISnackbarState {
+  message: string;
+  error: boolean;
+  show: boolean;
+}
+
+@Module({
+  namespaced: true,
+  dynamic: true,
+  store: Store,
+  name: 'SnackbarModule'
+})
+export class Snackbar extends VuexModule {
+  message = '';
+  error = false;
+  show = false;
+  @Action
+  success(message: string) {
+    this.context.commit('SET_ERROR', false);
+    this.context.commit('SET_MESSAGE', message);
+    this.context.commit('SET_VISIBILITY', true);
+  }
+  @Action
+  failure(message: string) {
+    this.context.commit('SET_ERROR', true);
+    this.context.commit('SET_MESSAGE', message);
+    this.context.commit('SET_VISIBILITY', true);
+  }
```

Comment:
I believe `this.context.commit('SET_ERROR', false);` can be replaced with `this.SET_ERROR(false);`, etc. This is how it is done in `server.ts` and it seems to function fine.